app/lib/admin/backend.dart

```diff
@@ -271,6 +277,117 @@ class AdminBackend {
     });
   }
 
+  Future<void> removePackage(String packageName) async {
```

Comment:
Please add a brief documentation, especially concerning the immutability of the call.